### ISSUE TITLE
perlex with coord mask & conf_mask

### ISSUE THREAD

**Issue by speeding-motor** [experiencor/basic-yolo-keras#453]
**conf_mask why not use y_true mask:**
if y_true[i,4]=1 , it means there have a box, and conf_mask[i]=1,

why still need to compute the IOU between y_pred with y_true box , and then judge the conf_mask through :`conf_mask= conf_mask + tf.to_float(best_ious < 0.6) * (1 - y_true[..., 4])`

**Comment by speeding-motor** [comment]
how to understand the conf_mak  here, 
`loss_conf = tf.reduce_sum(tf.square(true_box_conf - pred_box_conf) * conf_mask)`

can you please help me ...thank you very much
